tests/std/tests/P1222R4_flat_set/test.cpp

```diff
@@ -132,9 +119,97 @@ void test_constructors() {
     flat_set<int> a{};
     a = {1, 7, 7, 7, 2, 100, -1};
     assert_all_requirements_and_equals(a, {-1, 1, 2, 7, 100});
+    assert_all_requirements_and_equals(flat_set<int>(a, allocator<int>{}), {-1, 1, 2, 7, 100});
+    assert_all_requirements_and_equals(flat_set<int>(std::move(a), allocator<int>{}), {-1, 1, 2, 7, 100});
     flat_multiset<int> b{};
     b = {1, 7, 7, 7, 2, 100, -1};

```

Comment:
This was passing because the a `flat_set` was constructed from `initializer-list` and then `operator=(flat_set&&)` was called. Now it's `operator=(initializer_list)`